pom.xml

```diff
@@ -382,6 +382,18 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>travis</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.eluder.coveralls</groupId>
+            <artifactId>coveralls-maven-plugin</artifactId>
+            <version>4.3.0</version>
```

Comment:
can you replace the version with a variable? I'll be easier for child projects to select the version they want to use

Reply:
Certainly sir, coming up!